pilot/pkg/model/proxy_view.go

```diff
@@ -48,7 +48,7 @@ func newProxyView(node *Proxy) ProxyView {
 
 	// Restrict the view to the requested networks.
 	return &proxyViewImpl{
-		visible: sets.New(node.Metadata.RequestedNetworkView...).Insert(identifier.Undefined),
+		visible: sets.New[string](node.Metadata.RequestedNetworkView...).Insert(identifier.Undefined),
```

Comment:
using type inference is preferred

Reply:
I use GoLand programming, GoLand can't infer the type, because `RequestedNetworkView` is defined as `StringList` type, not directly `[]string`.

This change is so that GoLand does not prompt for syntax errors.